engine/src/test/java/org/hibernate/search/test/bridge/MultiFieldMapBridge.java

```diff
@@ -34,4 +36,8 @@ private void setField(String fieldPrefix, String key, String value, Document doc
 		luceneOptions.addFieldToDocument( fieldPrefix + "." + key, value, document );
 	}
 
+	@Override
+	public void configureFieldMetadata(String name, FieldMetadataBuilder builder) {
+		builder.field( "value", FieldType.OBJECT );
```

Comment:
This should not be a hardcoded constant (`"value"`), but simply `name`. I know this won't change anything in practice, but it's just in case people copy/paste this code.